src/components/Popups/PopupList.tsx

```diff
@@ -1,10 +1,10 @@
 import React from 'react';
 import isEmpty from 'lodash/isEmpty';
 import { FormattedMessage } from 'react-intl';
-import ItemList from '../../ItemList/ItemList';
-import messages from '../messages';
-import PopupBase from '../PopupBase';
-import { Options, PopupReference } from '../Popper';
+import ItemList from '../ItemList/ItemList';
```

Comment:
Can this component be exported/imported via an index file?

Reply:
Sorry it's my fault. I created an index file in the ItemList folder but forgot to take advantage of it when importing the `ItemList`. @ConradJChan  Can you correct it for me? Thanks